bazel/repositories.bzl

```
@@ -60,3 +61,10 @@ def foreign_cc_repositories():
         strip_prefix = "ws-protocol-releases-cpp-v" + WS_PROTOCOL_VERSION,
         sha256 = "1c7d7b874f2e20d841cd04391d9d0be507ccb75b22f84b65a0fc61a30ac30651",
     )
+
+    git_repository(
+        name = "stdexec",
+        commit = "dc8f1688bad95eff99699c775251ee47e82bd4f1",
+        remote = "git@github.com:NVIDIA/stdexec.git",
+        build_file = "//bazel/foreign_cc:stdexec.BUILD",
+    )
```

`http_archive` is faster and more reliable
```suggestion
    STDEXEC_VERSION = "dc8f1688bad95eff99699c775251ee47e82bd4f1",
    http_archive(
        name = "stdexec",
        urls = ["https://github.com/NVIDIA/stdexec/archive/{version}.zip".format(version = STDEXEC_VERSION)],
        strip_prefix = "stdexec-" + STDEXEC_VERSION,
        build_file = "//bazel/foreign_cc:stdexec.BUILD",
        sha256 = "....",
    )
```